OCPBUGS-14057: Add runbooks description for prometheus alerts which ingress operator provides (#166)
openshift/runbooks: alerts/cluster-ingress-operator/HAProxyReloadFail.md

# HAProxyReloadFail

## Meaning

This alert fires when HAProxy fails to reload its configuration, which will
result in the router not picking up recently created or modified routes.

> ## Impact

Reviewer: Add:

Warning only.

The router won't pick up recently created or modified routes. This may cause
an outage for critical applications.

## Diagnosis

Check the router logs:
```sh
oc logs <router pod> -n openshift-ingress
```

> Check if any recently added configuration in the haproxy config via ingress
> controller CR caused the issue.

Reviewer: ```suggestion
Check if any recently added configuration in the haproxy config via ingress
controller spec caused the issue.
```

> 

Reviewer: Tell them how to access the container and check the haproxy.config for issues.

## Mitigation

> Try to fix the configuration of the haproxy via ingress controller CR on the
> basis of the output of the logs.

Reviewer: ```suggestion
Try to fix the configuration of the haproxy by editing the ingress controller spec.
```
